src/legacy/core_plugins/interpreter/server/lib/route_expression/thread/index.js

```diff
@@ -27,9 +27,9 @@ const workerPath = resolve(__dirname, 'babeled.js');
 const heap = {};
 let worker = null;
 
-export function getWorker() {
+export function getWorker(registeredPaths) {
   if (worker) return worker;
-  worker = fork(workerPath, {});
+  worker = fork(workerPath, [ JSON.stringify(registeredPaths) ]);
```

Comment:
I think it would be better to pass this via the env

```js
fork(workerPath, [], {
  env: {
    ...process.env,
    interpreterPluginPaths: JSON.stringify(registeredPaths)
  }
})
```